### ISSUE TITLE
memory leak ? 

### ISSUE THREAD

**Issue by 123mitnik** [mirror-media/plate-vue-dfp#32]
where do you remove the event listener - googletag.pubads().addEventListener('slotRenderEnded', this.dfpOptions['afterEachAdLoaded'].bind(this))

**Comment by chiangkeith** [comment]
@123mitnik You're right, there's nowhere to remove this event listener actually, because DFP doesnt provide any way to remove it...

**Comment by 123mitnik** [comment]
yes... there is no way inside DFP . i can't find a way to circumvent this . it is causing huge memory leak in my app
